src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+function convertToNumber(SalaryString) {
```

Comment:
it is customary to write parameter names with a lowercase letter
```suggestion
function convertToNumber(salaryString) {
```